### ISSUE TITLE
Full size bar lines being rendered poking up

### ISSUE THREAD

**Issue by Nokel81** [gregorio-project/gregorio#1586]
gregorio verison: 6.0.0

When rendering a full bar `(:)` or double bar `(::)` the line seems to extend a little above the top horizontal line.

![image](https://github.com/gregorio-project/gregorio/assets/8225332/bb736be4-24e3-43fe-b38f-4af99cd1a60d)
![image](https://github.com/gregorio-project/gregorio/assets/8225332/8acdbd60-126e-450a-89e6-707324858a67)

**Comment by rpspringuel** [comment]
While I think the issue is being exaggerated in the first image by the pdf renderer, I can confirm that the second image I see the same thing as the second image when I zoom in sufficiently.  Unfortunately my fontforge installation is currently not working so I can't actually try to fix this at the moment.

**Comment by Nokel81** [comment]
Yeah I agree that the above issue is probably the PDF renderer (which in this case is Adobe IIRC) i think it might be fixed if we fix the other issue too.

If this is a font issue, I have an installation of fontforge up and running, how would I go about fixing that?

**Comment by rpspringuel** [comment]
There are two possibilities that need to be explored:

1) the bar line is being placed too high. Checking this doesn’t involve any font changes, but rather playing with the way glyphraisevalue is calculated for the bar to get a slightly smaller number. If you can lower the bar line glyph without it sticking out below the bottom line of the staff, then this would be the solution. 

2) the bar line glyph is too tall. Fixing this is what would involve editing the fonts to shorten the bar line glyph. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
